src/App.tsx

```diff
@@ -1,57 +1,22 @@
-import React from 'react';
+import React, { useState, FC } from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Dropdown } from './components/Dropdown/Dropdown';
 
-export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+export const App: FC = () => {
+  const [selectedPerson, setSelectedPerson] = useState<Person>();
+
+  const personInfo = `${selectedPerson?.name}
+   (${selectedPerson?.born}
+     =
+    ${selectedPerson?.died})`;
 
   return (
     <main className="section">
-      <h1 className="title">
-        {`${name} (${born} = ${died})`}
-      </h1>
-
-      <div className="dropdown is-active">
-        <div className="dropdown-trigger">
-          <input
-            type="text"
-            placeholder="Enter a part of the name"
-            className="input"
-          />
-        </div>
-
-        <div className="dropdown-menu" role="menu">
-          <div className="dropdown-content">
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Bernard Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter Antone Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Haverbeke</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-link">Pieter de Decker</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Petronella de Decker</p>
-            </div>
-
-            <div className="dropdown-item">
-              <p className="has-text-danger">Elisabeth Hercke</p>
-            </div>
-          </div>
-        </div>
-      </div>
+      {selectedPerson
+        ? <h1 className="title">{personInfo}</h1>
+        : <h1 className="title">No selected person</h1>}
+      <Dropdown setSelectedPerson={setSelectedPerson} delayTime={1000} />
```

Comment:
no need to pass it as prop, just create a constant for it in Dropdown component
```suggestion
      <Dropdown setSelectedPerson={setSelectedPerson} />
```